model-java/src/main/java/org/jboss/pnc/causeway/model/ProductReleaseImportResult.java

```diff
@@ -1,31 +1,30 @@
 package org.jboss.pnc.causeway.model;
 
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
-/**
- * Created by jdcasey on 2/9/16.
- */
 public class ProductReleaseImportResult
```

Comment:
We actually need non-final fields with getters/setters if we expect to use a library like Jackson to deserialize this without custom serialization logic. That may seem like a secondary concern right now, but having a real client api can make it MUCH easier to test...and it's also a nice way for anyone else to pick up and use our service. (Especially think about the point where we're not just going imports, but also allowing PNC to query brew via our service.)